### DISCUSSION TITLE
What's wrong with the smooth moving of Arcade Physics? Jittering, laggy moving

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [photonstorm/phaser#6294] [Q&A]
I look at this example of Arcade Physics moving:
https://labs.phaser.io/view.html?src=src/physics/arcade/body%20controls.js&v=3.60.0-beta.14
(I set collideWorldBounds on)
Every 5-7 seconds I see a little random jump, I believe it's called jittering.

Also I've done some comparison "ArcadePhysics vs Sprite.x moving" here: https://codepen.io/GDv777/pen/mdxpPqd
Top object looks more laggy than the another one. Is that some problem with ArcadePhysics, with Phaser3 or with our browsers? 
We test it on many browsers, the result is always the same, we are all see that top object has much more jittering than just moving it with "sprite.x" method.

Also I've noticed that MatterJS (in Phaser) has more smoothing movement than Arcade.

**Comment by samme** [comment]
A little jitter is expected because the Arcade Physics step delta is different from the main game loop delta. You can mitigate it by increasing the physics FPS (e.g. `{ fps: 300 }`) or eliminate it by using a variable step (`{ fixedStep: false }`).

https://codepen.io/samme/pen/MWXqyYE?editors=0010
